src/systemtest/resources/30daysFromUntilDate/expected/reposense_testrepo-Beta_master/authorship.json

```diff
@@ -456,11 +457,13 @@
       "April0616": 1,
       "-": 52,
       "nbr": 11
-    }
+    },
+    "binaryFileAuthors": []
   },
   {
     "path": "src/main/resources/view/PersonInfoOverview.fxml",
     "fileType": "fxml",
+    "isBinary": false,
```

Comment:
Instead of creating a new field `isBinary`, is it possible to just set the `fileType` to be binary? Then can just check if `fileType == 'Binary'` to see if the file is binary

In this way the json files won't have to add another extra field to each file object

Reply:
I believe that we should separate it.
Say we have something like this.

![image](https://user-images.githubusercontent.com/32454748/75621172-6427b000-5bcc-11ea-973c-65c798da5f9d.png)

What should happen if I uncheck `txt`? Should `picture.txt` disappear as well?
Similarly, what should happen if I uncheck `Binary`?

But no matter how we want it to behave currently, separating `FileType` and `isBinary` would be helpful if we were to change its behaviour in the future.